projects/igniteui-angular/src/lib/directives/focus-trap/focus-trap.directive.spec.ts

```diff
@@ -0,0 +1,185 @@
+import { Component, ViewChild } from '@angular/core';
```

Comment:
general note about the tests:
In focus trap directive test you should test the directive. Create test component with several inputs and buttons. Put all them in a div, and add `igxFocusTrap` directive on the div. Then test focus is trapped in the div.
All the dialog related test should be added in dialog component tests.